Blizzard.IO.RabbitMQ/Helpers.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Blizzard.IO.RabbitMQ.Entities;
+using System.Collections.Generic;
+
+namespace Blizzard.IO.RabbitMQ
+{
+    public static class Helpers
```

Comment:
Myabe make this class internal.
I dont think youll even call this in the tests
```suggestion
    internal static class Helpers
```

Reply:
Maybe call this class in a better name I think even Utilities is better.
```suggestion
    public static class Utilities
```
if you accept it dont forget to change the file name

Reply:
I need those in the tests too

Reply:
I didn't change the access modifier to `internal` but I renamed the class to `Utilities`